src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -656,6 +656,7 @@ public FunctionInfo GetFunctionInfo(Syntax.CallNode node)
         // form of expression with personal info removed,
         // suitable for logging the structure of a formula.
         private string _expressionAnonymous;
+        private string _simpleAnonymizerExpression;

```

Comment:
Suggestion: _simpleAnonimizedExpression (anonymizer => anonymized), as it stores the anonymized expression itself, not the logic to anonymize it

Reply:
yes, typo